.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,38 @@
+name: Build and Test
+run-name: Build and Test 🚀
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-22.04
+    container: fedora:38
+    strategy:
+      fail-fast: false
+      matrix:
+        buildtype:
+          - Debug
+          - Release
```

Comment:
`bpfilter`'s `CMakeLists.txt` uses lowercase build types, you can do it here too.